custom_dc/env.list

```diff
@@ -0,0 +1,36 @@
+# API key for accessing the base Data Commons API.
+DC_API_KEY=
+# API key for accessing Maps and Places APIs for exploration tools.
+MAPS_API_KEY=
+# Use local SQLite as the database.
+USE_SQLITE=true
+# Input directory in Google Cloud Storage or local drive, containing files to be processed.
+INPUT_DIR=
+# The data path of the files stored in Google Cloud Storage, in the form gs://<BUCKET>/<FOLDER>/.
+# GCS_DATA_PATH overrides OUTPUT_DIR.
+GCS_DATA_PATH=
```

Comment:
@keyurva This doesn't seem to take effect for stats.main.
Does it need any other config?

Reply:
This is used by a [script](https://github.com/datacommonsorg/website/blob/83be489138c1c91463b5d80ff0bb3d757b3504dc/build/web_compose/compose.sh#L26) that is run in the docker image, it is not used by the simple importer.

This script can map `GCS_DATA_PATH` / `OUTPUT_DIR` to the `output_dir` flag when calling `stats.main`.

Reply:
Set OUTPUT_DIR to the GCS_DATA_PATH if set.